server2/Arcadia.Assistant/Arcadia.Assistant.SickLeaves.Contracts/ISickLeaves.cs

```diff
@@ -17,6 +18,8 @@ public interface ISickLeaves : IService
     {
         Task<SickLeaveDescription[]> GetCalendarEventsAsync(EmployeeId employeeId, CancellationToken cancellationToken);
 
+        Task<Dictionary<EmployeeId, SickLeaveDescription[]>> GetCalendarEventsCollectionAsync(EmployeeId[] employeeIds, CancellationToken cancellationToken);
```

Comment:
Bad naming.

Reply:
Can you recomend correct format?

Reply:
Technically, dictionary is also a collection, but your name doesn't make clear what will be returned. For example, `GetCalendarEventsByEmployeeMap` looks better for me.

Reply:
Good. Thanx.

Reply:
Done